src/webui/webapplication.cpp

```diff
@@ -495,6 +502,16 @@ Http::Response WebApplication::processRequest(const Http::Request &request, cons
             throw UnauthorizedHTTPError();
         }
 
+        // reverse proxy resolve peer address
+        if (m_isReverseProxyEnabled)
+        {
+            m_clientAddress = resolvePeerAddress();
+        }
+        else
+        {
+            m_clientAddress = m_env.clientAddress;
+        }
```

Comment:
Maybe move all the logic above into `resolvePeerAddress()`? And rename it to `resolveClientAddress()`?

Reply:
Yes `Client` is more consistent than `Peer`.